### ISSUE TITLE
Adding empty commandbuf to history

### ISSUE THREAD

**Issue by MoonPadUSer** [blue-nebula/base#130]
You can navigate through your `commandbuf` history already using `UP` and `DOWN` already, which is great.
But I'd like to have the ability to press `DOWN` once more when being at the most recent entry of the history to get an empty `commandbuf`

**Comment by TheAssassin** [comment]
I am strongly opposed to this. IMO it is more than enough to be able to press escape and then `T` (or whatever chat bind you have) again. I am a heavy shell user, and I would hate such a feature because the bottom line should always be what you typed last. Clearing is easy in shells too, just press `Ctrl+C`. I know that I would be likely to lose what I typed, and this would be highly annoying. I'd even go so far to consider this an antifeature.

TL;DR: just press `ESC` and then chat again, no need for additional complexity.

**Comment by MoonPadUSer** [comment]
I mean .... I could also just make it so it automatically adds this hline temporarily to the history, and it'd be deleted once you closed the console, so basically, only the "submitted" ones would be kept
